From 0e44c819b72dfad40a7f9eea6067f6060fa9c35b Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 19 Aug 2015 10:03:50 -0400
Subject: [PATCH] Ticket 47831 - remove debug logging from retro cl

Description:  Instrumented debug logging was accidentally left in the source.
              This logging is being removed.

https://fedorahosted.org/389/ticket/47931

Reviewed by: mreynolds

(cherry picked from commit db7153f89bf3dda935e6ef4f175697bda32fe720)
(cherry picked from commit 1781280f133c4877f83949400294641a558f5406)
---
 ldap/servers/plugins/retrocl/retrocl_po.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/ldap/servers/plugins/retrocl/retrocl_po.c b/ldap/servers/plugins/retrocl/retrocl_po.c
index f689373..d9f4e6d 100644
--- a/ldap/servers/plugins/retrocl/retrocl_po.c
+++ b/ldap/servers/plugins/retrocl/retrocl_po.c
@@ -157,14 +157,11 @@ write_replog_db(
     int	err = 0;
     int ret = LDAP_SUCCESS;
     int	i;
-    int mark = 0;
 
     if (!dn) {
         slapi_log_error( SLAPI_LOG_PLUGIN, RETROCL_PLUGIN_NAME, "write_replog_db: NULL dn\n");
         return ret;
     }
-    mark = (post_entry && retrocl_entry_in_scope(post_entry));
-    slapi_log_error( SLAPI_LOG_FATAL, RETROCL_PLUGIN_NAME, "post in scope (%d)\n",mark);
 
     if (post_entry){
         if(!retrocl_entry_in_scope(log_e) && !retrocl_entry_in_scope(post_entry)){
-- 
1.9.3

